From a427c99331b28e7e45f931578c448d2f547e2f90 Mon Sep 17 00:00:00 2001
From: snodlop <samkahn02@yahoo.com>
Date: Wed, 14 Jan 2026 20:23:44 -0800
Subject: [PATCH 07/16] roxanne update 1

just gave roxanne more mons, these are not set in stone, will edit later
---
 src/data/trainers.party | 36 ++++++++++++++++++++++++++++++++++--
 1 file changed, 34 insertions(+), 2 deletions(-)

diff --git a/src/data/trainers.party b/src/data/trainers.party
index 48ee05bd04..843a94e24c 100644
--- a/src/data/trainers.party
+++ b/src/data/trainers.party
@@ -4806,14 +4806,46 @@ IVs: 12 HP / 12 Atk / 12 Def / 12 SpA / 12 SpD / 12 Spe
 - Rock Throw
 - Rock Tomb
 
-Geodude
+Bonsly
 Level: 12
 IVs: 12 HP / 12 Atk / 12 Def / 12 SpA / 12 SpD / 12 Spe
-- Tackle
+- Flail
+- Defense Curl
+- Rock Throw
+- Sucker Punch
+
+Cranidos
+Level: 13
+IVs: 12 HP / 12 Atk / 12 Def / 12 SpA / 12 SpD / 12 Spe
+- Headbutt
 - Defense Curl
 - Rock Throw
 - Rock Tomb
 
+Lunatone
+Level: 14
+IVs: 12 HP / 12 Atk / 12 Def / 12 SpA / 12 SpD / 12 Spe
+- Ancient Power
+- Stored Power
+- Icy Wind
+- Calm Mind
+
+Tirtouga
+Level: 14
+IVs: 12 HP / 12 Atk / 12 Def / 12 SpA / 12 SpD / 12 Spe
+- Smack Down
+- Aqua Jet
+- Protect
+- Whirlpool
+
+Nacli
+Level: 13
+IVs: 24 HP / 24 Atk / 24 Def / 24 SpA / 24 SpD / 24 Spe
+- Salt Cure
+- Rock Polish
+- Mud-Slap
+- Harden
+
 Nosepass @ Oran Berry
 Level: 15
 IVs: 24 HP / 24 Atk / 24 Def / 24 SpA / 24 SpD / 24 Spe
-- 
2.52.0.windows.1

